dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -172,7 +174,7 @@ private String getRepositoryUrl(String artifactId) {
 
   public class Version implements Comparable<Version> {
 
-    private String version;
+    public String version;
 
     public final String get() {
```

Comment:
Ok, my suggestion is to just annotate this method with `@JsonValue`. This will cause...